test/JDBC/expected/BABEL-SYS-DATABASES.out

```diff
@@ -18,19 +18,19 @@ GO
 SELECT name, compatibility_level, collation_name FROM sys.databases ORDER BY name;
```

Comment:
Is it possible to add these tests as prepare/verify/cleanup format? If so then we should do that instead of maintaining multiple files for possibly same test cases.

Reply:
Yea, I'll try to do taht

Reply:
Tried this a bit, but not sure if it's possible or a good idea since it enables multi-db and switching to single-db after. The issue with this test is that it switches back to single-db is causing an issue. I think we should keep the test as a separate file since it does test that sys.database displays multiple user databases in multi-db mode.

 Correct me if I'm wrong though with these following points on why prepare/verify/cleanup format doesn't work:

- If we keep it as is, (switch back to single-db) it will turn off multi-db option which would affect other upgrade tests in the upgrade test framework causing them to fail as multiple dbs are used. 

- If we switch back to single-db, it might cause problems for other tests in the regular JDBC since they run under single-db mode. 

- If we choose to keep it separate by switching back to single-db in prepare/verify/cleanup format in the input folder, but leave it as multi-db for all the version upgrade path, then it would fail when 14_latest schedule runs for the next version since we would switch back to single-db mode when 14_latest -> 14_latest+1 (making the test only effective for a regular non upgrade paths which is why we should keep it as is).

Reply:
Okay, Let it be as it is for the time being.